modules/bridge/src/main/ts/ephox/bridge/components/menu/ContextMenu.ts

```diff
@@ -1,13 +1,13 @@
 import { SeparatorMenuItemSpec } from '../../api/Menu';
+import { CommonMenuItemSpec } from './CommonMenuItem';
 
-export interface ContextMenuItem {
-  text: string;
```

Comment:
This shouldn't have been removed as it's not the same. Previously `text` was mandatory, but with this change it'll be optional.